cmd/cloud/dbt.go

```diff
@@ -0,0 +1,154 @@
+package cloud
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	cloud "github.com/astronomer/astro-cli/cloud/deploy"
+	"github.com/astronomer/astro-cli/cloud/deployment"
+	"github.com/astronomer/astro-cli/config"
+	"github.com/pkg/errors"
```

Comment:
```suggestion
	"errors"
```
same argument as before regarding pkg/errors being not actively developed